style.css

```diff
@@ -0,0 +1,113 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  width: 100%;
+  height: 618px;
+  left: 100%;
+  top: 0;
+  font-size: 1.5rem;
+  color: #344563;
+  line-height: 2.4rem;
+  font-weight: 400;
+  font-family: "Poppins", sans-serif;
+  margin: 0;
+}
+
+h1,
+h2,
+h3 {
+  font-weight: 700;
+  color: #172b4d;
+  margin-bottom: 15px;
+}
+
+h1 {
+  font-size: 40px;
+}
+
+h2 {
+  font-size: 32px;
+}
+
+h3 {
+  font-size: 13px;
+}
+
+.header-part {
+  display: flex;
+  margin: 26px;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 20px;
+  height: 40px;
+}
+
+.header-part a {
+  text-decoration: none;
+  color: #6070ff;
+  font-weight: 700;
+  font-size: 18px;
+}
+
+nav .fa {
+  color: #6070ff;
+}
+
+.headline-part {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: #fff url(./images/header-shapes\ mobile.png);
```

Comment:
- [ ] The background image which you have in this `headline-part` section is not consistent with the provided design template. Kindly take a look at what it should look like below 👇 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/163984294-b360dadb-9e78-4a9c-a9c1-2fa1e0190eb8.png)


### Figma Design
![image](https://user-images.githubusercontent.com/46839548/163984173-520a0810-f85b-4657-a0fb-4b41c89f8ccb.png)